parcels/particleset.py

```diff
@@ -9,6 +9,7 @@
 import bisect
 from collections import Iterable
 from datetime import timedelta as delta
+import pandas as pd
```

Comment:
So we're using `pandas` now? Is that really necessary? That's a pretty hefty requirement. I think you said on Slack you wanted to stick with `datetime` and `np`? I'd agree with that

Reply:
The only reason for pandas was that we could use float number of seconds.
(also, str(np.datetime64)) is quite ugly in pset.show(), so I think I'll clean it.
But yes, I'll change this (there are still a few things I wanna clean)

Reply:
@willirath, yes so pandas is available, but loading a module to have in one single location (the title of a plot)
2000-01-01 03:04:04.57
instead of 
2000-01-01 03:04:04
Is it worth it?